zproject/backends.py

```diff
@@ -178,6 +178,14 @@ def process_do_auth(self, user_profile, *args, **kwargs):
 
         return redirect_and_log_into_subdomain(realm, full_name, email_address)
 
+    def auth_complete(self, *args, **kwargs):
+        # type: (*Any, **Any) -> Optional[HttpResponse]
+        try:
+            # Call the auth_complete method of BaseOAuth2 is Python Social Auth
+            return super(SocialAuthMixin, self).auth_complete(*args, **kwargs)  # type: ignore
+        except AuthFailed:
+            return None
```

Comment:
For the time being I have made the code to return `None` to keep it consistent with how we handle `AuthFailed` exception in GitHub backend. Due to this the user will be redirected to 'login' page.